### PR TITLE
perf: dedupe consecutive identical frames

### LINKED ISSUE #58: Smarter de-duplication of frames
We currently do some binary search on progress in "fast" mode, but we could do better by also de-duping identical frames in all cases, not just fast.

### PR DESCRIPTION
**Pull request by patrickhulce**
fixes #58 

doesn't result in a huge win, ~1-2% since most frames have some small differences

### CODE REVIEW

File: src/speed-index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const imageSSIM = require('image-ssim');
 /* BEGIN FAST MODE CONSTANTS - See function doc for explanation */
 const fastModeAllowableChangeMax = 5;
 const fastModeAllowableChangeMedian = 3;
-const fastModeAllowableChangeMin = 1;
```
Comment (patrickhulce) [inline src/speed-index.js:8]: drive-by fix since I thought this is what we decided already :)

doesn't affect any of the tests but prevents skipping very large gaps

**Review by paulirish — APPROVED** [review]
